frontend/src/pages/DifficultySelect.js

```diff
@@ -11,12 +11,10 @@ import { useEffect, useState, useCallback, useContext } from "react";
 import { io } from "socket.io-client";
 import { useNavigate } from "react-router-dom";
 import { context } from "../context";
+import DifficultyCard from "../components/DifficultyCard";
+import TimedLoader from "../components/TimedLoader";
 
-const Difficulty = {
-  Easy: "easy",
-  Medium: "medium",
-  Hard: "hard",
-};
+const Difficulties = ['Easy', 'Medium', 'Hard']
```

Comment:
The database queries rely on sending a lowercase version of these difficulties, please change the mapping here to lowercase and supply it to handleDifficultyButton